app.js

```diff
@@ -9,12 +9,15 @@ var arrayHelperFunctions = require('./custom_modules/helpers/module_ArrayHelperF
 var objectHelperFunctions = require('./custom_modules/helpers/module_ObjectHelperFunctions');
 var routingHelperFunctions = require('./custom_modules/helpers/module_RoutingHelperFunctions');
 
+var blockchainSetup = require('./custom_modules/blockchain/module_BlockchainSetup');
```

Comment:
fair enough is following the conventions from yoeman gen (which is need of updating) but I'll rearrange for this